src/plat/imx8m-evk/overlay-imx8mq-evk.dts

```diff
@@ -26,4 +26,9 @@
 		status = "disabled";
 	};
 
+    guestvm@b0000000 {
```

Comment:
I'm going to drop this change as a carve out isn't needed if this isn't going to be used in camkes-vm projects yet.